translate_api/project.py

```diff
@@ -0,0 +1,27 @@
+from docx import Document
+from googletrans import Translator
+
+def read_doc(file_path):
```

Comment:
@Aaron-Yue-Zhao I don’t like the param names for this fun, could you please change them?